.jscsrc

```diff
@@ -4,8 +4,9 @@
     "maximumLineLength": null,
 
     "disallowMultipleVarDecl": "strict",
-    "disallowSpacesInAnonymousFunctionExpression": null,
-    "disallowSpacesInFunctionExpression": null,
+    "disallowSpacesInAnonymousFunctionExpression": false,
```

Comment:
I think all of these options needs to be `null`, the Appveyor build is failing on this (it passes locally and on Travis though, which is super weird).